framework/source/class/qx/bom/Cookie.js

```diff
@@ -63,7 +63,7 @@ qx.Bootstrap.define("qx.bom.Cookie",
         end = document.cookie.length;
       }
 
-      return unescape(document.cookie.substring(len, end));
+      return decodeURI(document.cookie.substring(len, end));
     },
 
```

Comment:
I think this may break existing cookies. The escape() function encodes all ascii characters other than letters/numbers, including '+' and '&'. If those characters were escaped()'ed, then decodeURI() will not yield the original result. I believe, however, that decodeURIComponent() will, but we should confirm, in a test, that at least every ascii character, and preferably other latin characters as well, are properly decoded from existing cookies... or decide that we just don't care. (I think we do care, though, until a major release.)